queries/odin/highlights.scm

```diff
@@ -0,0 +1,293 @@
+; Preprocs
+
+[
+  (calling_convention)
+  (tag)
+] @preproc
+
+; Includes
+
+[
+  "import"
+  "package"
+] @include
+
+; Keywords
+
+[
+  "foreign"
+  "using"
+  "struct"
+  "enum"
+  "union"
+  "defer"
+  "cast"
+  "transmute"
+  "auto_cast"
+  "map"
+  "bit_set"
+  "matrix"
+] @keyword
+
+[
+  "proc"
+] @keyword.function
+
+[
+  "return"
+  "or_return"
+] @keyword.return
+
+[
+  "distinct"
+  "dynamic"
+] @storageclass
+
+; Conditionals
+
+[
+  "if"
+  "else"
+  "when"
+  "switch"
+  "case"
+  "where"
+  "break"
+  (fallthrough_statement)
+] @conditional
+
+((ternary_expression
+  [
+    "?"
+    ":"
+    "if"
+    "else"
+    "when"
+  ] @conditional.ternary)
+  (#set! "priority" 105))
```

Comment:
Hm, can't this be done in a way that doesn't require tweaking priority?

Reply:
It's because of ":" in punctuation as well, I could move it below punctuation, but I like where it is now